src/utils/json.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// JSON utilities.
+
+/**
+ * Custom replacer function for JSON.stringify to handle JS objects that
+ * cannot otherwise be serialized.
+ */
+export function replacer (key, value) {
+  if (value instanceof Map) {
+    return { _jsonType: 'Map', _val: Array.from(value) }
```

Comment:
Out of interest, is there any convention / library for doing this?

Seems like something that would be useful, e.g. for Python <-> JS stuff.